xml/sle_update_online.xml

```diff
@@ -722,7 +720,7 @@ Continue? [y/n/? shows all options] (y):</screen>
     </para>
 <screen>&prompt.sudo;snapper rollback</screen>
     <para>
-     Reboot afterward. On the boot screen, choose the default boot entry to
+     Reboot the machine. In the boot screen, choose the default boot entry to
```

Comment:
"On"->"in" imo makes it worse. Might make sense to ask a native speaker about this (again?).